### ISSUE TITLE
add term request "teat skin"

### ISSUE THREAD

**Issue by stheil15** [obophenotype/uberon#2353]
**Preferred term label:**
"teat skin" from cow.

**Synonyms**

**Definition (free text, please give PubMed ID)**
In figure 1.1.
[publication](https://www.semanticscholar.org/paper/The-bovine-teat-canal%3A-Its-role-in-pathogen-and-of-Smolenski/c85c749158d33d751e115cf920f6e56e635212be)

**Parent term (use https://www.ebi.ac.uk/ols/ontologies/uberon)**
[skin epidermis](http://purl.obolibrary.org/obo/UBERON_0001003) ?

**Your nano-attribution (ORCID)**
0000-0001-7361-4159

**Comment by stheil15** [comment]
FoodOn term request :
https://github.com/FoodOntology/foodon/issues/193

**Comment by shawntanzk** [comment]
assigning myself, but its low priority for me, happy for someone to take over if its more urgently needed.

**Comment by shawntanzk** [comment]
I was wondering if http://purl.obolibrary.org/obo/UBERON_0005083 'nipple sheath' would work?
This is the definition: A circular ingrowth of the epidermis around the region of the mammary sprout
and the logical axioms are: 
<img width="775" alt="Screenshot 2022-06-23 at 17 11 04" src="https://user-images.githubusercontent.com/76212760/175262926-60eebc69-47b6-41f5-9587-744911b8c609.png">
Given a nipple is often called a teat when referring to non-humans, I think the nipple sheath is basically the nipple skin which is basically a teat skin
Can we therefore add both nipple skin and teat skin as exactly synonyms instead?

**Comment by stheil15** [comment]
The "nipple sheath" is a structure in formation as referred by the "mammary sprout" term.

The teat skin can be defined in this [figure](https://www.semanticscholar.org/paper/The-bovine-teat-canal%3A-Its-role-in-pathogen-and-of-Smolenski/c85c749158d33d751e115cf920f6e56e635212be/figure/0) and publication.

**Comment by shawntanzk** [comment]
I'm trying to understand the difference between nipple sheath and teat skin so we can model this properly:
1) Is it right to say that nipple skin = teat skin (one being in human, one in animals?) 
2) Is the nipple sheath part of the nipple skin? 

While the figure is useful, we still would need a textual definition and a logical definition that matches. If I were to create a term, it would be: 

Label: nipple skin
Def: A skin epidermis that is part of the nipple 
Synonym: teat skin [exact]
Comment: A nipple is often called a teat when referring to non-humans
Equivalent: Skin epidermis and ("part of" some nipple) 

The above would automatically place nipple sheath as a nipple skin - but that would be the only class under it which is generally frowned upon (what other classes should be there if so?) 

Thanks

**Comment by stheil15** [comment]
1. I would say it is right. Even if i found it weird, as in french teat = trayon and it really refers to animals that you milk (traire in french).
2. I do not really know if "nipple sheath" is part of nipple skin. In the way i understand it, refers to a structure before the formation of the teat.

I agree with the term definition but I do not understand why it places nipple sheath as a nipple skin ? (sorry)

**Comment by shawntanzk** [comment]
Current nipple sheath has the following logical axioms:
<img width="775" alt="Screenshot 2022-06-23 at 17 11 04" src="https://user-images.githubusercontent.com/76212760/175262926-60eebc69-47b6-41f5-9587-744911b8c609.png">
If I make an equivalent class (which makes sense given the name)
"Skin epidermis and ("part of" some nipple)" for nipple skin - a reasoner would reason that nipple sheath is a type of nipple skin 
I think I'm a little stuck here now - perhaps @ddooley might have a suggestion here?

**Comment by RDruzinsky** [comment]
Is the difference between a nipple and a teat made clear anywhere in the
ontology? The definition of nipple (
http://purl.obolibrary.org/obo/UBERON_0002030) is "Projection of skin
containing the outlets for 15-20 lactiferous ducts arranged cylindrically
around the tip.) This definition is 1) anthropocentric, because the number
of ducts varies across species, and 2) is not the definition of a teat. In
a nipple, the individual lactiferous ducts open directly to the outside
world. Most of the outlets are found in the nipple but some are in the
areola. In a teat the lactiferous ducts converge on a large lactiferous sac
at the base of the teat and there is a single, large duct that runs through
the teat with an outlet at the end of the teat. While many lay sources
treat nipple and teat as synonyms, mammalogists do not.


Robert E. Druzinsky, Ph.D.
Clinical Associate Professor
Dept. of Oral Biology
College of Dentistry
University of Illinois at Chicago
801 S. Paulina
Chicago, IL 60612
***@***.***

Office: 312-996-0406
Lab: 312-996-0629
Website: www.peerj.com/RobertDruzinsky

**Comment by shawntanzk** [comment]
> Is the difference between a nipple and a teat made clear anywhere in the ontology

It isn't, teat isnt a term in the ontology - it is currently a narrow synonym of nipple (which makes sense after what you mentioned). I think perhaps it might be sensible to create a new term "teat" and define the distinction in the ontology. 
I am happy to add it to the ontology, however, I do not have the capacity to really read into this (this is very far out of scope for me/my funding - mostly doing this as I am shepherding the tickets in Uberon and I thought this would be an easy add until I tried to do it and notice that it had the same subclass logical axioms as nipple sheath and realise i didnt know what the difference was).

Depending on the urgency on this term, I can add "teat skin" first and leave a comment that teat needs to be formalise to make proper logical axioms (so basically the only axiom will be that it is_a "skin epidermis". However the proper thing to do would be to formalise "teat" and maybe have a pairwise homology relation (in_historical_homology_relationship, or whatever the accurate version of that is) with nipple (I assume evolutionarily speaking, they came from the same ancestory?) The other thing I would like, if possible, is to have taxon restrictions on teat - I would need to know where in the evolutionary branch is it teat instead of nipple. But I need direction input on all the above and I'm guessing that will take quite awhile. 

That being said what @RDruzinsky wrote is a really good starting point :) 

> In a nipple, the individual lactiferous ducts open directly to the outside
> world. Most of the outlets are found in the nipple but some are in the
> areola. In a teat the lactiferous ducts converge on a large lactiferous sac
> at the base of the teat and there is a single, large duct that runs through
> the teat with an outlet at the end of the teat. While many lay sources
> treat nipple and teat as synonyms, mammalogists do not.

PS sorry for making a simple request so complicated - I honestly did not expect this too >.<

**Comment by stheil15** [comment]
Thanks everybody for the effort for defining this term.
@shawntanzk I did not expect this to be this complicated either.

@RDruzinsky definition is indeed a good starting point to define "teat" term. But I'm suprized that I can't find a clear definition since teat placement and size are a strong criteria of selection for cows, goats, ...

For the taxonomic restriction, I have found two mammals that are producing milk that do not have teat : Echidna and the Platypus. But for the distinction between nipple and teat I did not found anything revelant.
